### ISSUE TITLE
Problem with ListView templates in separate file

### ISSUE THREAD

**Issue by hannta** [jasonkneen/AlloyXL#2]
I have list templates in separate file like this:
```
<ListView>
  <Templates>
    <Require src="listTemplates" />
  </Templates>
  ...
</ListView>

```

```
// listTemplates.xml
<Alloy>
  <ItemTemplate name="normalItemTemplate">
  ...
  </ItemTemplate>
</Alloy>
```

This is causing crash:
> Uncaught TypeError: view.addEventListener is not a function

This happens on `view.addEventListener('postlayout', function(e)`

Im assuming it is safe to fist check existence of view addEventListener function (`(typeof view.addEventListener == 'function')`...), before trying to add event listener?

**Comment by jasonkneen** [comment]
Good one. I'll fix this.

**Comment by jasonkneen** [comment]
Should be fixed now on 0.9.7
